content/en/Getting started/local.md

```diff
@@ -135,11 +111,22 @@ You can test your service by visiting [http://localhost:8080/hello](http://local
 
 - SSH into the container
 ```bash
-opta shell -c hello.yaml
+opta shell --local -c hello.yaml
 
 root@staging-hello-k8s-service-57d8b6f478-vwzkc:/#
 ```
 
+- View container logs of your application
+```
+opta logs --local -c hello.yaml
+```
+
+- View Kubernetes for your Application's deployment Events
+```
+opta events --local -c hello.yaml
```

Comment:
this command is currently hidden, do we want to make it visible then?

Reply:
It works great so no reason to hide it imo, but @NitinAgg do you have another opinion?